docs/user/01-manager.md

```diff
@@ -2,14 +2,16 @@
 
 ## Module lifecycle
 
-The module on its own ships a single component only, namely the Telemetry Manager, which is described by a custom resource of type Telemetry. The Telemetry Manager implements the Kubernetes controller pattern and manages the whole lifecycle of all other components relevant to this module. The Telemetry Manager looks out for the Kubernetes LogPipeline, TracePipeline, and, in the future, MetricPipeline resources created by the user. With these, the user describes in a declarative way what data of a signal type to collect and where to ship it.
+Kyma's Telemetry module on its own ships a single component only, namely the Telemetry Manager. Telemetry Manager is a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that is described by a custom resource of type Telemetry. The Telemetry Manager implements the Kubernetes controller pattern and manages the whole lifecycle of all other components relevant to the Telemetry module.
+The Telemetry Manager looks out for the user-created Kubernetes resources: LogPipeline, TracePipeline, and, in the future, MetricPipeline. In these resources, you specify what data of a signal type to collect and where to ship it.
 If the Telemetry Manager detects a configuration, it rolls out the relevant components on demand.
 
 ![Manager](./assets/manager-lifecycle.drawio.svg)
```

Comment:
You've removed all verb -s endings, but I'd keep them. The verbs describe actions performed by the subjects expressed in singular forms.

Reply:
You're right! Just like in all other diagrams. I added them again. Thanks for noticing!